### PR TITLE
View cleanup

### PR DESCRIPTION
**Pull request by SlipperyJ**

### CODE REVIEW

File: app/assets/stylesheets/reviews.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the reviews controller here.
```
Comment (Bennell) [inline app/assets/stylesheets/reviews.scss]: Not in use so remove this file

File: app/assets/javascripts/reviews.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Bennell) [inline app/assets/javascripts/reviews.coffee]: Not in use so remove this file

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
+class ReviewsController < ApplicationController
```
Comment (Bennell) [inline app/controllers/reviews_controller.rb]: To be honest I'd remove almost everything in this controller that we're not using right now. Add functionality as required. Definitely clear out all the comments.

File: app/controllers/wheelchairs_controller.rb [hunk 4]
```diff
@@ -1,2 +1,74 @@
 class WheelchairsController < ApplicationController
+  before_action :set_wheelchair, only: [:show, :edit, :update, :destroy]
+
```
Comment (Bennell) [inline app/controllers/wheelchairs_controller.rb]: same as above

File: app/helpers/reviews_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module ReviewsHelper
```
Comment (Bennell) [inline app/helpers/reviews_helper.rb]: This file isn't it use so remove it

File: app/controllers/reviews_controller.rb (hunk 3 above)
Comment (xiy) [inline app/controllers/reviews_controller.rb]: No need to remove **everything** as this is just a simple CRUD app. Though I would strip out the JSON rendering parts as we're never going to need that.

**Comment by xiy** [comment]
:+1:

**Pull request merged**